test/stmt/statements.swift

```diff
@@ -207,9 +207,15 @@ func IfStmt1() {
 
 func IfStmt2() {
   if 1 > 0 {
-  } else // expected-error {{expected '{' after 'else'}}
+  } else // expected-error {{expected '{' or 'if' after 'else'}}
   _ = 42
 }
+func IfStmt3() {
+  if 1 > 0 {
+  } else 1 < 0 { // expected-error {{expected '{' or 'if' after 'else'; did you mean to write 'if'?}}
```

Comment:
Mind adding `{{9-9= if}}`?

Reply:
Oh, is that the way to test fixits? I assume `9` is the column number?

Reply:
Yeah, usually, we don't add `FixCode` test case for testing fix-its like this. But is I think it's OK to add it :)